.github/workflows/tsbench-coverage.yaml

```diff
@@ -0,0 +1,204 @@
+# A job to inspect code coverage of current TSBench benchmarks, and also run
+# them with assertions enabled to have more test coverage.
+name: TSBench Code Coverage
+"on":
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
Do we need to schedule this action? Shouldn't this scheduled run produce the same output as the last run on `main`?